balena/models/application.py

```diff
@@ -881,3 +914,300 @@ def get_dashboard_url(self, app_id):
             self.settings.get('api_endpoint').replace('api', 'dashboard'),
             '/apps/{app_id}'.format(app_id=app_id)
         )
+
+
+class ApplicationInvite():
+    """
+    This class implements application invite model for balena python SDK.
+
+    """
+
+    def __init__(self):
+        self.base_request = BaseRequest()
+        self.settings = Settings()
+        self.config = Config()
+        self.auth = Auth()
+        self.release = Release()
+        self.RESOURCE = 'invitee__is_invited_to__application'
+
+    def get_all(self):
+        """
+        Get all invites.
+
+        Returns:
+            list: list contains info of invites.
+            
+        Examples:
+            >>> balena.models.application.invite.get_all()
+            [{'id': 5860, 'message': 'Test invite', 'invitee': {'__id': 2965, '__deferred': {'uri': '/resin/invitee(@id)?@id=2965'}}, 'is_created_by__user': {'__id': 5227, '__deferred': {'uri': '/resin/user(@id)?@id=5227'}}, 'is_invited_to__application': {'__id': 1681618, '__deferred': {'uri': '/resin/application(@id)?@id=1681618'}}, 'application_membership_role': {'__id': 2, '__deferred': {'uri': '/resin/application_membership_role(@id)?@id=2'}}, '__metadata': {'uri': '/resin/invitee__is_invited_to__application(@id)?@id=5860'}}]
+
+        """
+
+        return self.base_request.request(
+            self.RESOURCE, 'GET', endpoint=self.settings.get('pine_endpoint')
+        )['d']
+        
+    def get_all_by_application(self, app_id):
+        """
+        Get all invites by application.
+
+        Args:
+            app_id (str): application id.
+
+        Returns:
+            list: list contains info of invites.
+            
+        Examples:
+            >>> balena.models.application.invite.get_all_by_application(1681618)
+            [{'id': 5860, 'message': 'Test invite', 'invitee': {'__id': 2965, '__deferred': {'uri': '/resin/invitee(@id)?@id=2965'}}, 'is_created_by__user': {'__id': 5227, '__deferred': {'uri': '/resin/user(@id)?@id=5227'}}, 'is_invited_to__application': {'__id': 1681618, '__deferred': {'uri': '/resin/application(@id)?@id=1681618'}}, 'application_membership_role': {'__id': 2, '__deferred': {'uri': '/resin/application_membership_role(@id)?@id=2'}}, '__metadata': {'uri': '/resin/invitee__is_invited_to__application(@id)?@id=5860'}}]
+
+        """
+
+        params = {
+            'filter': 'is_invited_to__application',
+            'eq': app_id
+        }
+
+        return self.base_request.request(
+            self.RESOURCE, 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint')
+        )['d']
+        
+    def create(self, app_id, invitee, role_name=None, message=None):
+        """
+        Creates a new invite for an application.
+
+        Args:
+            app_id (str): application id.
+            invitee (str): the email/balena_username of the invitee.
```

Comment:
That's actually something we need to change in the JS SDK as well
```suggestion
            invitee (str): the email of the invitee.
```
See: https://github.com/balena-io/balena-api/blob/a0472380577b2fd516c3d64929fb42af86830b22/src/features/invitations/hooks/application.ts#L74-L76